apps/web/components/OrderReturnProductCard/OrderReturnProductCard.vue

```diff
@@ -67,33 +67,33 @@
           </li>
         </ul>
       </div>
-    </div>
-    <div class="md:mx-5 md:grid grid-cols-[1fr_1fr] self-end">
-      <UiQuantitySelector
-        :key="quantity"
-        @change-quantity="debounceQuantity"
-        :value="quantity"
-        :min-value="0"
-        :max-value="orderGetters.getItemQty(orderItem)"
-        class="mt-4 sm:mt-0 h-fit self-end mr-4"
-      />
+      <div class="md:mx-5 self-end flex justify-end">
+        <UiQuantitySelector
+          :key="quantity"
+          @change-quantity="debounceQuantity"
+          :value="quantity"
+          :min-value="0"
+          :max-value="orderGetters.getItemQty(orderItem)"
+          class="mt-4 sm:mt-0 h-fit self-end mr-4"
+        />
 
-      <div class="flex flex-col flex-1 justify-end">
-        <label>
-          <span class="pb-1 text-sm font-medium text-neutral-900"> {{ $t('returns.returnReason') }} </span>
-          <SfSelect
-            @update:model-value="changeReason($event)"
-            :model-value="String(returnReasonId)"
-            size="sm"
-            class="h-fit"
-            :placeholder="$t(`returns.selectReturnReason`)"
-          >
-            <option :value="null">— {{ $t('returns.selectReturnReason') }} —</option>
-            <option v-for="{ id, name } in returnReasons.reasons" :key="id" :value="id">
-              {{ name }}
-            </option>
-          </SfSelect>
-        </label>
+        <div class="flex flex-col flex-1 justify-end">
+          <label>
+            <span class="pb-1 text-sm font-medium text-neutral-900"> {{ $t('returns.returnReason') }} </span>
+            <SfSelect
+              @update:model-value="changeReason($event)"
+              :model-value="String(returnReasonId)"
+              size="sm"
+              class="h-fit py-[0.55rem]"
```

Comment:
is there a reason you use a custom padding value here?

Reply:
discussed in chat.

No other predefined classes would fit the height of the sibling, and also couldn't use flex as the return reason has a label on top.

Reply:
👍